plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -189,6 +199,24 @@ public OptionalLong getLimit()
         return sortOrder;
     }
 
+    @JsonIgnore // not needed on workers
+    public Set<SchemaTableName> getOtherReferencedTables()
+    {
+        return otherReferencedTables;
+    }
+
+    @JsonIgnore
```

Comment:
add comment here too

Reply:
adding `// not a getter` here

Reply:
i made the first field regular JsonProperty
(alternatively i'd need to make it `Optional` and this would bloat the code.)